support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -202,6 +232,22 @@ const shorterBoxMargin = css`
 	}
 `;
 
+const validPaymentMethods = [
+	query.product !== 'Contribution' && countryGroupId === 'EURCountries' && Sepa,
```

Comment:
Maybe check for AplePay (safari browser or IOS with ApplePay setup)

Reply:
I'm going to bring this into another PR as it's a bit more complicated (added to #5722)